include/picongpu/particles/ParticlesInit.kernel

```diff
@@ -331,10 +331,12 @@ namespace picongpu
 
                             numParsPerCellCtx[ idx ]--;
                             if( numParsPerCellCtx[ idx ] > 0 )
-                                atomicExch(
+                                nvidia::atomicAllExch(
```

Comment:
QUESTION: 
Is using the nvidia namespace for an general (alpaka) routine not counter-intuitive? 
This routine seems to be defined for various `acc` not only for `gpus`.

Reply:
The nvidia namespace is within PMacc but PMacc used alpaka under the hut even if the namespace is called nvidia.

Reply:
But that is confusing 😉 - will this be changed in the future?